src/GitHub.App/ViewModels/Dialog/Clone/RepositoryCloneViewModel.cs

```diff
@@ -240,6 +242,11 @@ string ValidatePathWarning(RepositoryModel repositoryModel, string path)
                 {
                     using (var repository = gitService.GetRepository(path))
                     {
+                        if (service.DestinationDirectoryEmpty(path))
```

Comment:
I'd be inclined to make this part of the `if` above, e.g.

```cs
if (service.DestinationDirectoryExists(path)) && service.DestinationDirectoryEmpty(path))
```

That way we don't need to create a repository object if the directory is empty.

Reply:
Thanks for pointing this out. I have corrected this issue.